app/_src/kubernetes-ingress-controller/guides/high-availability/fallback-config.md

```diff
@@ -0,0 +1,480 @@
+implementation details and provide an example scenario to demonstrate how it works in practice.
+
+{% include /md/kic/prerequisites.md release=page.release disable_gateway_api=false %}
+
+## Overview 
+
+{{site.kic_product_name}} in version 3.2.0 introduced the Fallback Configuration
+feature. It is designed to isolate issues related to individual parts
+of the configuration, allowing updates to the rest of it to proceed with no
+interruption. If you're using {{site.kic_product_name}} in a multi-team environment, the
+fallback configuration mechanism can help you avoid lock-ups when one team's
+configuration is broken.
+
+{:.note}
+> **Note:** The Fallback Configuration is an opt-in feature. You must
+> enable it by setting `FallbackConfiguration=true` in the controller's feature
+> gates configuration.
+
+## How it works
+
+{{site.kic_product_name}} translates Kubernetes objects it gets from the Kubernetes API and pushes
+the translation result via Kong’s Admin API to {{site.base_gateway}} instances. However, issues
+can arise at various stages of this process:
+
+1. Admission Webhook: Validates individual Kubernetes objects against schemas and basic rules.
+2. Translation Process: Detects issues like cross-object validation errors.
+3. Kong Response: Kong rejects the configuration and returns an error associated with a specific object.
+
+Fallback Configuration is triggered when an issue is detected in the 3rd stage and provides the following benefits:
+- Allows unaffected objects to be updated even when there are configuration errors.
+- Automatically builds a fallback configuration that Kong will accept without requiring user intervention by
+  either:
+  - Excluding the broken objects along with its dependants.
+  - Backfilling the broken object along with its dependants using the last valid Kubernetes objects' in-memory cache (if `--use-last-valid-config-for-fallback` flag is set).
+- Enables users to inspect and identify what objects were excluded from or backfilled in the configuration using
+  diagnostic endpoints.
+
+Below table summarizes the behavior of the Fallback Configuration feature based on the configuration:
+
+| `FallbackConfiguration` feature gate value | `--use-last-valid-config-for-fallback` flag value | Behavior                                                                                                                                             |
+|--------------------------------------------|---------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `false`                                    | `false`/`true` (has no effect)                    | The last valid configuration is used as a whole to recover (if stored).                                                                              |
```

Comment:
Should KIC throw an error if `--use-last-valid-config-for-fallback` is passed when `FallbackConfiguration` is false?

Reply:
Yeah, that makes sense. Posted a PR adding that: https://github.com/Kong/kubernetes-ingress-controller/pull/6168